src/olympia/amo/templates/amo/paginator.html

```diff
@@ -1,38 +1,36 @@
-{% if num_pages > 1 %}
```

Comment:
Those pagination changes are to move the `num_pages > 1` condition out of the template and in the parent where needed - because for the merged queue, we want to display the count from the pagination at all times, even when there is only a single page.